src/legacy/core_plugins/elasticsearch/lib/create_proxy.js

```diff
@@ -63,7 +62,7 @@ export function createProxy(server) {
           body
         }, { signal });
       } catch (error) {
-        throw handleESError(error);
```

Comment:
This was changed in a [previous PR](https://github.com/elastic/kibana/pull/43923) I did, but the `_search` behavior is only used in two places (courier and querying matching index patterns), and both rely on the behavior of not throwing, but simply returning the response from ES.